src/constants.js

```diff
@@ -152,3 +152,11 @@ export const devhubMapping = {
     nodejs: 'Node.js',
     java: 'Java',
 };
+
+// Twitch API constants
+export const TWITCH_API_ENDPOINT = 'https://api.twitch.tv/helix/';
+export const TWITCH_CLIENT_ID = '041r2glmgub2pt357ss0la44j2sz95';
+export const TWITCH_MDB_CHANNEL_ID = '467752938';
+export const TWITCH_headers = { 'Client-ID': TWITCH_CLIENT_ID };
```

Comment:
```suggestion
export const TWITCH_HEADERS = { 'Client-ID': TWITCH_CLIENT_ID };
```

small nit: This should also be in caps